src/kibana/components/vislib/visualizations/line_chart.js

```diff
@@ -71,10 +71,20 @@ define(function (require) {
       var xScale = this.handler.xAxis.xScale;
       var yScale = this.handler.yAxis.yScale;
       var ordered = this.handler.data.get('ordered');
-      var visibleRadius = 3;
       var touchableRadius = 12;
       var tooltip = this.tooltip;
       var isTooltip = this._attr.addTooltip;
+      var radii = _(data)
+        .map(function (series) { return _.map(series, function (point) { return point._input.z; }); })
+        .flatten()
+        .value();
+
+      radii = {
```

Comment:
Rather than redefining `radii` here, you could use `_.transform()` at the end of your chain. Especially since the two different versions of radii are of different types.